src/chordsymbol.ts

```diff
@@ -395,227 +222,123 @@ export class ChordSymbol extends Modifier {
-    const yoff = 0.25 * this.textFormatter.fontSizeInPixels;
-    let symIndex = 0;
-    for (symIndex === 0; symIndex < barIndex; ++symIndex) {
-      const symbol = this.symbolBlocks[symIndex];
-      symbol.xShift = symbol.xShift + xoff;
-      symbol.yShift = symbol.yShift - yoff;
-    }
-
-    for (symIndex = barIndex + 1; symIndex < this.symbolBlocks.length; ++symIndex) {
-      const symbol = this.symbolBlocks[symIndex];
-      symbol.xShift = symbol.xShift - xoff;
-      symbol.yShift = symbol.yShift + yoff;
-    }
-  }
-
-  updateKerningAdjustments(): void {
-    let accum = 0;
-    for (let j = 0; j < this.symbolBlocks.length; ++j) {
-      const symbol = this.symbolBlocks[j];
-      accum += this.getKerningAdjustment(j);
-      symbol.xShift += accum;
-    }
-  }
-
-  /** Do some basic kerning so that letter chords like 'A' don't have the extensions hanging off to the right. */
-  getKerningAdjustment(j: number): number {
-    if (!this.useKerning) {
-      return 0;
-    }
-    const currSymbol = this.symbolBlocks[j];
-    const prevSymbol = j > 0 ? this.symbolBlocks[j - 1] : undefined;
-    let adjustment = 0;
-
-    // Move things into the '/' over bar
-    if (
-      currSymbol.symbolType === SymbolTypes.GLYPH &&
-      currSymbol.glyph !== undefined &&
-      currSymbol.glyph.code === ChordSymbol.glyphs.over.code
-    ) {
-      adjustment += currSymbol.glyph.metrics.xShift;
-    }
-
-    if (
-      prevSymbol !== undefined &&
-      prevSymbol.symbolType === SymbolTypes.GLYPH &&
-      prevSymbol.glyph !== undefined &&
-      prevSymbol.glyph.code === ChordSymbol.glyphs.over.code
-    ) {
-      adjustment += prevSymbol.glyph.metrics.xShift;
-    }
-
-    // For superscripts that follow a letter without much top part, move it to the left slightly
-    let preKernUpper = false;
-    let preKernLower = false;
-    if (prevSymbol !== undefined && prevSymbol.symbolType === SymbolTypes.TEXT) {
-      preKernUpper = ChordSymbol.upperKerningText.some((xx) => xx === prevSymbol.text[prevSymbol.text.length - 1]);
-      preKernLower = ChordSymbol.lowerKerningText.some((xx) => xx === prevSymbol.text[prevSymbol.text.length - 1]);
-    }
-
-    const kerningOffsetPixels = ChordSymbol.kerningOffset * this.textFormatter.fontSizeInPixels;
-    // TODO: adjust kern for font size.
-    // Where should this constant live?
-    if (preKernUpper && currSymbol.symbolModifier === SymbolModifiers.SUPERSCRIPT) {
-      adjustment += kerningOffsetPixels;
-    }
-
-    if (preKernLower && currSymbol.symbolType === SymbolTypes.TEXT) {
-      if (currSymbol.text[0] >= 'a' && currSymbol.text[0] <= 'z') {
-        adjustment += kerningOffsetPixels / 2;
-      }
-      if (ChordSymbol.upperKerningText.some((xx) => xx === prevSymbol?.text[prevSymbol.text.length - 1])) {
-        adjustment += kerningOffsetPixels / 2;
-      }
-    }
-    return adjustment;
+    return ChordSymbol.subscriptOffset * Font.convertSizeToPixelValue(this.textFont?.size);
```

Comment:
See the other comment about optional chaining. Let's make sure we aren't accidentally passing `undefined` to Font.convertSizeToPixelValue().